example/tests/elm-verify-examples.json

```diff
@@ -6,5 +6,20 @@
     "Robustness",
     "Failing",
     "Todo"
-  ]
+  ],
+  "ignoreWarnings": {
+    "Mock.Foo.Bar.Moo": [
+      {
+        "name": "dontCare",
+        "ignore": [
+          "NoExampleForExposedDefinition"
```

Comment:
So verbose to write this by hand. Wonder if we could do some magic like adding the ability to modify this from the cli. Like `elm-verify-examples --ignore Mock.Foo.Bar.Moo`